tools/loadtesting/terraform/ecr.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_ecr_repository" "prometheus-to-cloudwatch" {
```

Comment:
Should we remove this since we won't be using it?

Reply:
We still want to ship some prometheus metrics to cloudwatch. We just don't care about HTTP metrics per se.